src/libraries/Microsoft.Extensions.Primitives/src/StringValues.cs

```diff
@@ -418,7 +418,7 @@ IEnumerator IEnumerable.GetEnumerator()
         /// Indicates whether the specified <see cref="StringValues"/> contains no string values.
         /// </summary>
         /// <param name="value">The <see cref="StringValues"/> to test.</param>
-        /// <returns>true if <paramref name="value" /> contains a single null or empty string or an empty array; otherwise, false.</returns>
+        /// <returns><see langword="true"/> if <paramref name="value"/> contains a single null or empty string or an empty array; otherwise, <see langword="false"/>.</returns>
```

Comment:
```suggestion
        /// <returns><see langword="true"/> if <paramref name="value"/> contains a single <see langword="null"/> or empty string or an empty array; otherwise, <see langword="false"/>.</returns>
```